public/locales/es/translation.json

```diff
@@ -0,0 +1,193 @@
+      "confidante": "Confidente",
+      "cooking_together": "Cocinar juntes",
+      "craft": "Artesanía",
+      "cuddles": "Cariños",
+      "cultivating_uniqueness": "Cultivar la singularidad",
+      "d_s": "D/s",
+      "dance": "Baile",
+      "datemate": "Cita",
+      "discussions": "Discusiones",
+      "embracing_change": "Aceptación del cambio",
+      "emergencies": "Emergencias",
+      "emotional_attraction": "Atracción emocional",
+      "emotional_exclusivity": "Exclusividad emocional",
+      "empathy": "Empatía",
+      "end_of_life_death": "Fin de la vida / Muerte",
+      "events": "Eventos",
+      "writing": "Escritura",
+      "executor_of_will": "Testamentarie",
+      "family": "Familia",
+      "family_sick_or_elderly": "Familia (enfermedad y vejez)",
+      "fetishes": "Fetiches",
+      "friend": "Amigue",
+      "friends": "Amigues",
+      "friendship": "Amistad",
+      "gaming": "Juegos",
+      "genitals": "Genitales",
+      "hand_holding": "Tomarse de la mano",
+      "health": "Salud",
+      "holiday": "Ir de vacaciones",
+      "hugs": "Abrazos",
+      "improv": "Improvisación",
+      "intellectual_discussions": "Discusiones intelectuales",
+      "kisses": "Besos",
+      "kissing": "Besarse",
+      "listening": "Escuchar",
+      "love_languages": "Lenguajes de amor",
+      "m_s": "M/s",
+      "marriage": "Matrimonio",
+      "martial_arts": "Artes marciales",
+      "massage": "Masaje",
+      "mentor_guide": "Mentore - Guía",
+      "music": "Música",
+      "nudity": "Desnudez",
+      "organization": "Organización",
+      "orgasms": "Orgasmos",
+      "partner": "Compañere",
+      "payments": "Pagos",
+      "plants": "Plantas",
+      "power_of_attorney": "Poder notarial",
+      "projects": "Proyectos",
+      "property": "Propiedad",
+      "reliability": "Fiabilidad",
+      "responsibilities": "Responsabilidades",
+      "routine": "Rutina",
+      "routines": "Rutinas",
+      "sadomasochism": "Sadomasoquismo",
+      "seen_together": "Vistos juntes",
+      "sexual_exclusivity": "Exclusividad sexual",
+      "shared_accounts": "Cuentas compartidas",
+      "shared_activities": "Actividades compartidas",
+      "shared_beliefs": "Creencias compartidas",
+      "shared_experiences": "Experiencias compartidas",
+      "shared_goals": "Metas compartidas",
+      "shared_interests": "Intereses compartidos",
+      "shared_ritual": "Ritual compartido",
+      "shared_values": "Valores compartidos",
+      "sharing_a_home": "Compartir un hogar",
+      "sharing_a_sleeping_space": "Compartir un espacio para dormir",
+      "sharing_beliefs": "Compartir creencias",
+      "sharing_meals": "Compartir comidas",
+      "platonic": "Platónico",
+      "comet": "Cometa",
+      "long_distance": "A larga distancia",
+      "sharing_values": "Compartir valores",
+      "shibari": "Shibari",
+      "social_media": "Redes sociales",
+      "sponsor_sponsee": "Patrocinadore - Padrine",
```

Comment:
No sería "apadrinade"? Yo entiendo padrino = sponsor

Reply:
I think i need to understand this flavour to translate it better. Because Apadrinar, comes from padre and here in neutral circles we are starting to use adre instead of p/madre.

Reply:
eg https://www.therecoveryvillage.com/treatment-program/aftercare/sponsor-sponsee-relationship/
It is a flavour to express a possibility of a power imbalance of dependence that is important to people to describe their relationship